src/test/java/api/support/fixtures/policies/PoliciesActivationFixture.java

```diff
@@ -0,0 +1,226 @@
+import static api.support.http.ResourceClient.forCampuses;
+import static api.support.http.ResourceClient.forFixedDueDateSchedules;
+import static api.support.http.ResourceClient.forInstitutions;
+import static api.support.http.ResourceClient.forLibraries;
+import static api.support.http.ResourceClient.forLoanPolicies;
+import static api.support.http.ResourceClient.forLocations;
+import static api.support.http.ResourceClient.forNoticePolicies;
+import static api.support.http.ResourceClient.forRequestPolicies;
+import static api.support.http.ResourceClient.forServicePoints;
+import static api.support.http.api.support.NamedQueryStringParameter.namedParameter;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import api.support.RestAssuredClient;
+import api.support.builders.LoanPolicyBuilder;
+import api.support.builders.NoticePolicyBuilder;
+import api.support.fixtures.CirculationRulesFixture;
+import api.support.fixtures.LoanPoliciesFixture;
+import api.support.fixtures.LocationsFixture;
+import api.support.fixtures.LostItemFeePoliciesFixture;
+import api.support.fixtures.NoticePoliciesFixture;
+import api.support.fixtures.OverdueFinePoliciesFixture;
+import api.support.fixtures.RequestPoliciesFixture;
+import api.support.fixtures.ServicePointsFixture;
+import api.support.http.QueryStringParameter;
+
+// It is delegated by lombok in APITests class
+@SuppressWarnings("unused")
+public final class PoliciesActivationFixture {
+  private static final Logger log = LoggerFactory.getLogger(PoliciesActivationFixture.class);
+
+  private final RestAssuredClient restAssuredClient;
+  private final LoanPoliciesFixture loanPoliciesFixture;
+  private final RequestPoliciesFixture requestPoliciesFixture;
+  private final NoticePoliciesFixture noticePoliciesFixture;
+  private final OverdueFinePoliciesFixture overdueFinePoliciesFixture;
+  private final LostItemFeePoliciesFixture lostItemFeePoliciesFixture;
+  private final CirculationRulesFixture circulationRulesFixture;
+  private final LocationsFixture locationsFixture;
+
+  public PoliciesActivationFixture() {
+    restAssuredClient = defaultRestAssuredClient();
+
+    loanPoliciesFixture = new LoanPoliciesFixture(forLoanPolicies(), forFixedDueDateSchedules());
+    requestPoliciesFixture = new RequestPoliciesFixture(forRequestPolicies());
+    noticePoliciesFixture = new NoticePoliciesFixture(forNoticePolicies());
+    overdueFinePoliciesFixture = new OverdueFinePoliciesFixture();
+    lostItemFeePoliciesFixture = new LostItemFeePoliciesFixture();
+    circulationRulesFixture = new CirculationRulesFixture(restAssuredClient);
+    locationsFixture = new LocationsFixture(forLocations(), forInstitutions(),
+      forCampuses(), forLibraries(), new ServicePointsFixture(forServicePoints()));
+  }
+
+  public PoliciesActivation.PoliciesActivationBuilder rollingDefaultPolicies() {
```

Comment:
I think this reads more naturally as `default rolling policies`, what do you think?

```suggestion
  public PoliciesActivation.PoliciesActivationBuilder defaultRollingPolicies() {
```

Reply:
Agree, renamed.